actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -52,5 +52,11 @@ class Railtie < Rails::Railtie # :nodoc:
 
       ActionDispatch.test_app = app
     end
+
+    initializer "action_dispatch.system_tests" do |app|
```

Comment:
👋Hello!

 By including the `app.routes.url_helpers` inside the test itself, if you have a route that starts with `test_`, minitest will try to call it as it thinks it's a test https://github.com/seattlerb/minitest/blob/ab39d35fb4e84eb866ed9c4ecb707cbf3889de42/lib/minitest/test.rb#L66

It can be surprising to user and can break if the route expect parameters

Reply:
Agree. We should include this routes in a object and use `method_missing` to delegate to them like we do in the integration tests. @Edouard-chin can you open a PR?